package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "body-parser": "^1.13.3",
+    "bootstrap": "3.3.6",
```

Comment:
This is fixed to this version. Do we want to instead accept any 3.x changes?  i.e. `"^3.3.6"`?  That would match what we do elsewhere, although we've also talked about changing to fixed versions (but more so for the jupyter libs)...